### PR TITLE
[DOCFIX] Update CLI docs related to job service

### PR DESCRIPTION
**Pull request by bradyoo**

### CODE REVIEW

**Comment by bradyoo** [comment]
jenkins, test this please.

**Comment by bradyoo** [comment]
jenkins, test this please

File: docs/en/basic/Command-Line-Interface.md [hunk 1]
```diff
@@ -139,6 +139,37 @@ $ ./bin/alluxio getConf --unit KB alluxio.user.block.size.bytes.default
...
+1571865684758
+1571865684757
+
+# Displays the status info for the specific job.
+$ ./bin/alluxio job stat -v 1571865684755
+ID: 1571865684755
+Status: COMPLETED
+Task 0
+	Status: COMPLETED
+```
```
Comment (calvinjia) [inline docs/en/basic/Command-Line-Interface.md]: Update this to reflect the change to display the job name?

**Comment by bradyoo** [comment]
Updated

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please.

**Pull request merged**